dpctl/tests/test_usm_ndarray_ctor.py

```diff
@@ -1435,6 +1435,17 @@ def test_astype_device():
     assert r.sycl_queue == q2
 
 
+def test_astype_gh_1926():
+    get_queue_or_skip()
+
+    x = dpt.ones(10_000)
```

Comment:
I think size=10 would be enough.
I used big size in my reproducer only to demonstrate performance impact. It is not needed for testing